package.json

```diff
@@ -9,6 +9,7 @@
     "test-int": "NODE_ENV=test mocha --require co-mocha 'index.spec.js'",
     "test-auth": "NODE_ENV=test mocha --require co-mocha 'lib/auth.spec.js'",
     "server": "cd ./server && node server.js",
```

Comment:
Good idea, can we just modify the existing "server" script instead of creating a new "server test"?